docs/mkdocs.yml

```diff
@@ -2,8 +2,35 @@
 
 site_name: Python on whales
 docs_dir: generated_sources
+repo_url: "https://github.com/gabrieldemarmiesse/python-on-whales"
 site_description: 'Python on whales docs'
 
+markdown_extensions:
+- pymdownx.snippets:
+    check_paths: true
+
+plugins:
+  - search
+  - autorefs
+  - mkdocstrings:
```

Comment:
couldn't find an easy way to do it as well. so the workaround i picked is to change the docstring to be google style and now the arguments are shown in table nicely. btw I need to add the material as the theme, because the table looks quite weird without theme ( word got cut into half and placed in a new line in the table ). with material theme, everything looks tidy now